packages/terra-core-docs/src/terra-dev-site/doc/list/example/DraggableListItem.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import List, { Item } from 'terra-list/lib/index';
+import { Placeholder } from '@cerner/terra-docs';
+import classNames from 'classnames/bind';
+import styles from './ListDocCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ListDraggableExample = () => (
+  <List role="listbox" aria-label="example-label" isDraggable>
+    <Item
+      key="1"
+      isSelectable
+      isSelected
+    >
+      <Placeholder title="List Item 1" className={cx('placeholder')} />
+    </Item>
+    <Item
+      key="2"
+      isSelectable
+    >
+      <Placeholder title="List Item 2" className={cx('placeholder')} />
+    </Item>
+    <Item
+      key="3"
+      isSelectable
+    >
+      <Placeholder title="List Item 3" className={cx('placeholder')} />
+    </Item>
+    <Item
+      key="4"
+      isSelectable
+    >
+      <Placeholder title="List Item 4" className={cx('placeholder')} />
+    </Item>
+  </List>
```

Comment:
can we update this example with realistic values instead of `list item 1` and aria-label `example-label`

Reply:
Changes Done. Thanks.